packages/openneuro-components/src/header/header.scss

```diff
@@ -125,6 +125,18 @@
       margin-top: auto;
       margin-bottom: 30px;
     }
+    .header-aggriate {
+      display: flex;
+      justify-content: space-between;
+      max-width: 90%;
+      .aggregate-count {
```

Comment:
Having the AggregateCount's styles come from the header component's styles makes it less modular. This also makes its individual Storybook entry less useful.

I think that there should be separate scss file for AggregateCount. And then we can pass in something like a `placement` prop whose values can be `'header'` or `'cube'` and then have a class for each placement. Something like `.header-aggregate-count` and `.cube-aggregate-count`